modules/gitops-enable-traffic-management-and-metric-plugins-in-argo-rollouts.adoc

```diff
@@ -0,0 +1,95 @@
+
+. Select *{gitops-title}* from the *Installed Operators*.
+
+. In the *Details* tab, under the *Provided APIs* section, click *Create instance* in the *RolloutManager* pane.
+
+. On the *Create RolloutManager* page, select the *YAML view* and edit the YAML.
++
+.Example adding the traffic management and metric plugins configuration in the `RolloutManager` CR
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: RolloutManager
+metadata:
+  name: argo-rollouts
+spec:
+  plugins:
+    trafficManagement:
+      - name: argoproj-labs/gatewayAPI #<1>
+        location: https://github.com/sample-metric-plugin #<2>
+    metric:
+      - name: argoproj-labs/sample-prometheus #<3>
+        location: https://github.com/sample-trafficrouter-plugin #<4>
+        sha256: dac10cbf57633c9832a17f8c27d2ca34aa97dd3d #<5>
+----
+<1> Specifies the name of the `trafficManagement` plugin.
+<2> Specifies the location of the `trafficManagement` plugin.
+<3> Specifies the name of the `metric` plugin.
+<4> Specifies the location of the `metric` plugin.
+<5> Optional: Specifies the SHA256 signature of the plugin binary, which is downloaded and installed by the Rollouts controller.
+
+. Click *Create*.
+
+. In the *RolloutManager* tab, under the *RolloutManagers* section, verify that the *Status* field of the RolloutManager instance shows as *Phase: Available*.
+
+. Verify that the traffic management and metric plugins are installed correctly by completing the following steps:
+.. In the *Administrator* perspective, click *Workloads* -> *ConfigMaps*.
+.. Click the *argo-rollouts-config* ConfigMap.
++
+As a result, the plugins defined in the `RolloutManager` CR are updated in the *argo-rollouts-config* `ConfigMap`.
++
+.Example updated traffic management and metric plugins in the *argo-rollouts-config* `ConfigMap`
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: argo-rollouts-config
+  namespace: argo-rollouts
+  labels:
+    app.kubernetes.io/component: argo-rollouts
+    app.kubernetes.io/name: argo-rollouts
+    app.kubernetes.io/part-of: argo-rollouts
+data:
+  metricPlugins: |
+      - name: "argoproj-labs/sample-prometheus" #<1>
+        location: https://github.com/sample-metric-plugin #<2>
+        sha256: dac10cbf57633c9832a17f8c27d2ca34aa97dd3d #<3>
+  trafficRouterPlugins: |
+    - name: argoproj-labs/gatewayAPI #<4>
+      location: https://github.com/sample-metric-plugin #<5>
+      sha256: "" #<6>
+    - name: argoproj-labs/openshift #<7>
+      location: file:/plugins/rollouts-trafficrouter-openshift/openshift-route-plugin #<8>
+      sha256: "" #<9>
+
+----
+<1> Specifies the name of the `metric` plugin.
+<2> Specifies the location of the `metric` plugin.
+<3> Specifies the sha256 signature of the `metric` plugin.
+<4> Specifies the name of the `trafficmanagement` plugin, which must match the name required by the plugin so that the configuration is recognized.
```

Comment:
Hi @Dhruv-Soni11 Sorry I missed this earlier, can you please remove this line -  `which must match the name required by the plugin so that the configuration is recognized`. I think this is not required, thanks to @varshab1210 for pointing this out

Reply:
Fixed.